pyfoxsi/src/pyfoxsi/response/response.py

```diff
@@ -37,15 +51,25 @@ def __init__(self):
         for missing_shell in missing_shells:
             self._eff_area_per_shell.drop(str(missing_shell), 1, inplace=True)
         # now add the effective area of all of the shells together
-        self.effective_area = pd.DataFrame({'module': self._eff_area_per_shell.sum(axis=1), 'total': self._eff_area_per_shell.sum(axis=1)})
+        self.optics_effective_area = pd.DataFrame({'module': self._eff_area_per_shell.sum(axis=1), 'total': self._eff_area_per_shell.sum(axis=1)})
+        self.effective_area = self.optics_effective_area
```

Comment:
This is weird.  Since `optics_effective_area` is an object, `effective_area` simply points to the same memory (there is no deep copy).  There is no difference between the two attributes.  When `_add_optical_path_to_effective_area()` – via `_set_default_optical_path()` – modifies `optics_effective_area`, `effective area` is also unavoidably modified.